README.md

```diff
@@ -212,7 +212,8 @@ assuming your machines are joined to a FreeIPA domain.
           - name: monger-cockpit
             dns: ['localhost', 'www.example.com']
             ca: ipa
-            group: cockpit-ws
+            # NOTE: omit group when using cockpit 330 and later (Fedora and EL10)
+            group: cockpit-ws  # or cockpit-wsinstance on newer cockpit versions
```

Comment:
This advice is wrong and even dangerous -- files should *never* be owned by cockpit-wsinstance. cockpit-tls needs to read the certificates, but the actual `cockpit-ws` units really shouldn't. There were only two situations:

 - with cockpit < 257 the certificate needed to readable by group `cockpit-ws`
 - after that, the certificate ownership does not matter any more (it's copied by a tiny root helper process), so you can share it with other services; but it should not be owned by system groups.

It also keeps setting the group by default, which has been obsolete for a loooong time. It's only necessary on RHEL 7.